zendesk/ticket.go

```diff
@@ -60,6 +61,12 @@ type Collaborator struct {
 	Email string `json:"email"`
 }
 
+type Requester struct {
+	LocaleID *int   `json:"locale_id"`
+	Name     string `json:"name"`
```

Comment:
we follow the pointer - omitempty convention. Read more about it [here](https://www.sohamkamani.com/blog/golang/2018-07-19-golang-omitempty/)